tests/integration/test_scpi_psu_agent_integration.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+
+import ocs
+from ocs.base import OpCode
+
+from ocs.testing import (
+    create_agent_runner_fixture,
+    create_client_fixture,
+)
+
+from integration.util import create_crossbar_fixture
+
+from socs.testing.device_emulator import create_device_emulator
+
+pytest_plugins = "docker_compose"
+
+wait_for_crossbar = create_crossbar_fixture()
+run_agent = create_agent_runner_fixture(
+    "../agents/scpi_psu/scpi_psu_agent.py",
+    "scpi_psu_agent",
+    args=["--log-dir", "./logs/"],
+)
+client = create_client_fixture("psuK")
+gpib_emu = create_device_emulator(
+    {
+        "*idn?": "(instance-id=psuK),,,",  # manufacturer, model, serial, firmware
```

Comment:
Here's an example response to `"*idn?"` from an actual device: `Keithley instruments, 2230G-30-1, 9203269, 1.16-1.04`. (Though really, since it's only printed, not a big deal what comes back from this in the test.)